scipy/_lib/_array_api.py

```diff
@@ -463,6 +464,11 @@ def scipy_namespace_for(xp: ModuleType) -> ModuleType | None:
 
     if is_cupy(xp):
         import cupyx  # type: ignore[import-not-found,import-untyped]
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
+            # Strange that this is needed. For now, filtering warnings
+            # to pass tests; we'll figure out a better strategy.
+            import cupyx.scipy.signal
```

Comment:
There is probably a better way to do this, but:
```python3
import cupyx
getattr(cupyx.scipy, 'special')  # ok
getattr(cupyx.scipy, 'signal')  # AttributeError: module 'cupyx.scipy' has no attribute 'signal'

import cupyx.scipy.signal
getattr(cupyx.scipy, 'signal')  # ok
```
For now, I was just trying to get this to work. Alternatives welcome.

Reply:
x-ref https://github.com/cupy/cupy/issues/8336. Hopefully CuPy will behave like SciPy at some point.